nixos/roles/prometheus/default-alerts.nix

```diff
@@ -115,18 +115,18 @@
       };
 
       TelegrafDown = {
-        expr = ''min(up{job=~"telegraf",type!='mobile'}) by (source, job, instance, org) == 0'';
```

Comment:
Isn't this mobile filter a quite generic thing? I am using this for laptop monitoring or in general devices that are allowed to go offline.